pms/paths/library-content-search.yaml

```diff
@@ -0,0 +1,201 @@
+get:
+  tags:
+    - Library
+  summary: Search Library Section Items
```

Comment:
Let's leave both the summary and the operation ID at just Search Library and searchLibrary

That should generate a more logical command verb structure.

Reply:
@LukeHagar  👍 done an done